lib/Doctrine/ORM/EntityNotFoundException.php

```diff
@@ -15,8 +15,8 @@ class EntityNotFoundException extends \RuntimeException implements ORMException
     /**
      * Static constructor.
      *
-     * @param string   $className
-     * @param string[] $id
+     * @param string $className
+     * @param array  $id
```

Comment:
Should be `mixed[]` then. We don't allow just hinting something as `array` anymore, and `EntityNotFoundException` can indeed be raised with any stringable identifier.